jetty-servlet/src/test/java/org/eclipse/jetty/servlet/ServletHolderTest.java

```diff
@@ -31,8 +35,20 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.junit.jupiter.api.Assertions.fail;
 
+import java.io.IOException;
+
 public class ServletHolderTest
 {
+    
+    public static class FakeServlet extends HttpServlet
+    {
+        @Override
+        protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException
+        {
+            super.doGet(req, resp);
```

Comment:
It would be better to either have nothing here or a minimal response generation. Calling super is confusing.

Reply:
Done